webtau-graphql/src/test/groovy/org/testingisdocumenting/webtau/graphql/GraphQLCoverageTest.groovy

```diff
@@ -34,13 +34,13 @@ class GraphQLCoverageTest {
     }
 
     @Test
-    void "should provide non covered operations"() {
-        coverage.recordOperation(validationResult('allTasks', GraphQLOperationType.QUERY))
-        coverage.recordOperation(validationResult('complete', GraphQLOperationType.MUTATION))
-
-        coverage.nonCoveredOperations().should == ['*name'     | '*type'    ] {
-                                                   ___________________________________________
-                                                    'taskById' | GraphQLOperationType.QUERY
-                                                  'uncomplete' | GraphQLOperationType.MUTATION }
+    void "should provide non covered queries"() {
+        coverage.recordQuery(validationResult('allTasks', GraphQLQueryType.QUERY))
+        coverage.recordQuery(validationResult('complete', GraphQLQueryType.MUTATION))
+
+        coverage.nonCoveredQueries().should == ['*name'     | '*type'    ] {
+                                                   ____________________________________
```

Comment:
shifted underscore

Reply:
Man, the table formatting always gets me.  We need to find a way to automate it as it's quite painful to maintain by hand.

Reply:
Totally agree. The best thing I know is custom IDEA formatter via plugin. Another best thing is to make webtau more popular and ask JetBrains to help.

Reply:
The first option is definitely faster :)